package dz2;

public class MyArraySizeException extends Exception {          //создаем свой класс, наследуемся от Эксепшен.
    public MyArraySizeException() {                            // конструкторы бывают разные
        super("Размер массива неверный. Введите размер 4 на 4");
    }
}


// данный конструктор Создает новое исключение с прописываемым сообщением. Причина не инициализируется
// и впоследствии может быть инициализирована вызовом initCause.
//по умолчанию конструктор